Bearded.UI/Controls/implementations/ListControl.cs

```diff
@@ -265,6 +274,9 @@ private void updateContentLimits()
 
         private void removeCellsUpwards()
         {
+            if (itemSource == null)
+                throw new InvalidOperationException("The ItemSource is not set.");
```

Comment:
I think my comments are getting repetitive and boil down to this: Should we throw in private methods like this at all? It feels smelly to me. We should write our code so that it won't call private methods in invalid circumstances, instead of checking everywhere I think. (I'll stop adding the same comment to all the below cases.)

Reply:
:) I'll fix a problem of a comment in the entire file. I might miss it if the same thing happends in a different file, so a reminder there is nice to have, but yeah, I'll change it here as well (if we want to change this - see my answer on the other comment).